docs/corectl.md

```diff
@@ -13,9 +13,10 @@ corectl [flags]
 ### Options
 
 ```
-  -h, --help      help for corectl
-  -t, --traffic   Log JSON websocket traffic to stdout
-  -v, --verbose   Logs extra information
+  -h, --help               help for corectl
+  -j, --json               Set logging format to JSON
+  -l, --log-level string   Set logging level, one of; TRACE, DEBUG, INFO, WARN, ERROR, FATAL and PANIC (default "INFO")
+  -t, --traffic            Log JSON websocket traffic to stdout
```

Comment:
Maybe we can include traffic if log level is debug? One less flag to keep track of that way.

Reply:
Good point 👍